Service.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using EnumColors;
+
+namespace Task1_GUI
+{
+    internal class Service
+    {
+        public struct HigherCard
+        {
+            public static int nominal_of_HigherCard;
```

Comment:
Please, read the naming convention again. For public methods/fields/variables we should use PascalCase.